src/core/referencingfeaturelistmodel.h

```diff
@@ -70,14 +84,14 @@ class ReferencingFeatureListModel : public QAbstractItemModel
     QgsFeature feature() const;
 
     /**
-     * The relation connectiong the parent feature with the children in this model
+     * The relation connection to the parent feature with the children in this model
```

Comment:
```suggestion
     * The relation connecting the parent feature with the children in this model
```
Not sure about it. Feel free to accept or discard.